src/bindings/g_object/value.cr

```diff
@@ -38,6 +41,12 @@ module GObject
       when UInt8         then LibGObject.g_value_set_uchar(ptr, value)
       when GLib::Variant then LibGObject.g_value_set_variant(ptr, value)
       when ParamSpec     then LibGObject.g_value_set_param(ptr, value)
+      when Enum
+        if value.class._is_flags_enum?
+          LibGObject.g_value_set_flags(ptr, value)
+        else
+          LibGObject.g_value_set_enum(ptr, value)
+        end
```

Comment:
Correct me if I'm wrong, but I believe you added this change due to the use of GValue to read/write properties, it's a good change, but would be nice to go in a separate commit with their own set of tests.

Reply:
I think this change is small enough to be inside this PR.